Tools/Scripts/run-api-tests

```diff
@@ -146,9 +146,13 @@ def parse_args(args):
                              help='Passes that environment variable to the tests (--additional-env-var=NAME=VALUE)'),
 
         optparse.make_option('--remote-layer-tree', action='store_true', default=False,
-                             help='Use the remote layer tree drawing model (macOS non-WebKitLegacy only)'),
+                             help='Use the remote layer tree drawing model (macOS WebKit2 only)'),
+        optparse.make_option('--no-remote-layer-tree', action='store_true', default=False,
```

Comment:
Because this `--no` pattern was so common, I built an small argument parser that generalizing it:

```
from webkitcorepy import arguments
...
optparse.make_option(
    '--remote-layer-tree', '--no-remote-layer-tree',
    dest='remote-layer-tree', default=True,
    help='...',
)
```
Not sure it makes sense here because you're ultimately forwarding the argument to the test binary, but wanted to call this out in case it was helpful.

Reply:
In this case, we have to support both --remote-layer-tree and --no-remote-layer-tree as well as not specifying either. That third state is different from specifying either so we can't quite use it here.

Reply:
If you set `default=None`, that would give you the tri-state.

But, similar to my recent comment in #12161, I forgot that this code is the older `optparse` instead of `argparse`, so `webkitcorepy`'s `arguments` won't work, this turns out to be a bad suggestion.